randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -545,9 +549,6 @@ Extra - room_id: [14]
   * Layers: default
   * L0 Hatch to Cheddar Bay/Door to Aquarium Shaft West
   * Extra - door_idx: (33,)
-  > Door to Worst Room in the Game
```

Comment:
my phrasing in the note was bad.
Secret message jr should stay here. The grav+SJ should move onto this connection.

Reply:
is there any actual point to having it be a certain connection?